README.md

```diff
@@ -1,27 +1,7 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/kkmann/workshop-r-swe/HEAD?urlpath=rstudio) [![Join the chat at https://gitter.im/workshop-r-swe/community](https://badges.gitter.im/workshop-r-swe/community.svg)](https://gitter.im/workshop-r-swe/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-# title
+# GSWEP4R Workshop
 
+Sources and materials for the *"Good Software Engineering Practice for R Packages"*" workshop.
 
-## How this repository is structured
-
-...
-
-## Where to start
-
-1. Fork the repository https://github.com/kkmann/simulatr to your github account
-2. Add collaborators to the repository!
-2. Clone the repository into your working environment (RStudio & git need to be installed). Otherwise, you can click on the "launch binder" badge or [here](https://mybinder.org/v2/gh/kkmann/workshop-r-swe/HEAD?urlpath=rstudio) to open a (free) RStudio instance in the cloud. This instance is transient and will close after 10 minutes of inactivity. No data are saved between sessions.
-3. Create a branch
-4. Make some changes 
-5. Commit the changes to your branch
-6. Push your branch to your fork of the repository (requires a github personal access token)
-8. Open a pull request and wait for the checks to run through (careful, the badges in your fork need to be adjusted to the new repo url to work properly)
-9. Have a collaborator review the code and approve the pull request
-10. Merge back into main
-10. Celebrate and check out the updated documentation after a few minutes
-
-## Useful references
-
-* [first steps with git & github by RStudio](https://www.youtube.com/watch?v=jN6tvgt3GK8)
-* [RStudio cheatsheets/ Git & GitHub](https://github.com/rstudio/cheatsheets/blob/main/git-github.pdf)
+* [Website of the ASA Biopharmaceutical Section Software Engineering Working Group](https://rconsortium.github.io/asa-biop-swe-wg/)
```

Comment:
```suggestion
* [Website of the Software Engineering Working Group](https://rconsortium.github.io/asa-biop-swe-wg/)
```
maybe we slowly omit the ASA part... hoping that soon it is also EFSPI etc. and the name is long enough already